pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -415,6 +417,22 @@ func (dr *DashboardServiceImpl) SaveDashboard(ctx context.Context, dto *dashboar
 
 	return dash, nil
 }
+func (dr *DashboardServiceImpl) GetSoftDeletedDashboard(ctx context.Context, orgID int64, uid string) (*dashboards.Dashboard, error) {
+	return dr.dashboardStore.GetSoftDeletedDashboard(ctx, orgID, uid)
+}
+
+func (dr *DashboardServiceImpl) RestoreDashboard(ctx context.Context, orgID int64, dashboardUID string) error {
+	return dr.dashboardStore.RestoreDashboard(ctx, orgID, dashboardUID)
```

Comment:
In this particular case, the final goal of this flag is to be removed in the near future and leave the feature always turned on. 
I'm thinking the feature flag as a way to enable/disable the feature from the client perspective (API and frontend). Should we add it at all levels? Like the service layer, store layer and others?

Reply:
my personal preference would be to check the flag also in the service level but in case we won't I would suggest adding a comment highlighting that is not supposed to be called if the flag is not set

Reply:
Added the flag check